webhook_testing.go

```diff
@@ -0,0 +1,402 @@
+		</dispute>
+		`,
+	)
+}
+
+func (g *WebhookTestingGateway) disbursementXML(id string) string {
+	return g.sub(
+		id,
+		`
+		<disbursement>
+			<id>{{ $.ID }}</id>
+			<transaction-ids type="array">
+				<item>afv56j</item>
+				<item>kj8hjk</item>
+			</transaction-ids>
+			<success type="boolean">true</success>
+			<retry type="boolean">false</retry>
+			<merchant-account>
+				<id>merchant_account_token</id>
+				<currency-iso-code>USD</currency-iso-code>
+				<sub-merchant-account type="boolean">false</sub-merchant-account>
+				<status>active</status>
+			</merchant-account>
+			<amount>100.00</amount>
+			<disbursement-date type="date">2014-02-10</disbursement-date>
+			<exception-message nil="true"/>
+			<follow-up-action nil="true"/>
+		</disbursement>
+		`,
+	)
+}
+
+func (g *WebhookTestingGateway) disbursementExceptionXML(id string) string {
+	return g.sub(
+		id,
+		`
+		<disbursement>
+			<id>{{ $.ID }}</id>
+			<transaction-ids type="array">
+				<item>afv56j</item>
+				<item>kj8hjk</item>
+			</transaction-ids>
+			<success type="boolean">false</success>
+			<retry type="boolean">false</retry>
+			<merchant-account>
+				<id>merchant_account_token</id>
+				<currency-iso-code>USD</currency-iso-code>
+				<sub-merchant-account type="boolean">false</sub-merchant-account>
+				<status>active</status>
+			</merchant-account>
+			<amount>100.00</amount>
+			<disbursement-date type="date">2014-02-10</disbursement-date>
+			<exception-message>bank_rejected</exception-message>
+			<follow-up-action>update_funding_information</follow-up-action>
+		</disbursement>
+	`,
+	)
+}
+
+func (g *WebhookTestingGateway) partnerMerchantConnectedXML(id string) string {
+	return `
+	<partner-merchant>
+		<merchant-public-id>public_id</merchant-public-id>
+		<public-key>public_key</public-key>
+		<private-key>private_key</private-key>
+		<partner-merchant-id>abc123</partner-merchant-id>
+		<client-side-encryption-key>cse_key</client-side-encryption-key>
+	</partner-merchant>
+	`
+}
+
+func (g *WebhookTestingGateway) partnerMerchantDisconnectedXML(id string) string {
+	return `
+	<partner-merchant>
+		<partner-merchant-id>abc123</partner-merchant-id>
+	</partner-merchant>
+	`
+}
+
+func (g *WebhookTestingGateway) partnerMerchantDeclinedXML(id string) string {
+	return `
+	<partner-merchant>
+		<partner-merchant-id>abc123</partner-merchant-id>
+	</partner-merchant>
+	`
+}
+
+func (g *WebhookTestingGateway) accountUpdaterDailyReportXML(id string) string {
+	return `
+	<account-updater-daily-report>
+		<report-date type="date">2016-01-14</report-date>
+		<report-url>link-to-csv-report</report-url>
+	</account-updater-daily-report>
+	`
+}
+
+func (g *WebhookTestingGateway) sub(id string, xmlTemplate string) string {
+	var b bytes.Buffer
+	tmpl, err := template.New("").Parse(xmlTemplate)
+	err = tmpl.Execute(&b, struct{ ID string }{ID: id})
+	if err != nil {
+		panic(fmt.Errorf("creating xml template: " + err.Error()))
+	}
+	return b.String()
+}
```

Comment:
Any reason the contents of this file (`webhook_testing.go`) aren't part of the `webhook_testing_gateway.go` file?